### ISSUE TITLE
Mongo query with regex doesn't use /m modifier

### ISSUE THREAD

**Issue by dj-hedgehog** [balderdashy/sails-mongo#240]
I'm using MongoDB with Sails.
My Sails-query translates to the following:
`db.post.find( { body:  {"$regex" : /^.*twitter.*$/i }}).`

This query finds only posts which have 'twitter' in their body-field but don't have newlines. It won't find a match if a backslash is present.
When I alter it like this, it works as its supposed to.

`db.post.find( { body:  {"$regex" : /[^].*twitter.*$/im }}).`

I created a PR altering the behaviour of like and contains like this (not startsWith and endsWith)
https://github.com/balderdashy/sails-mongo/pull/242

**Comment by sailsbot** [comment]
Thanks for posting, @dj-hedgehog.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
